runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -195,14 +172,10 @@ def _check_for_child_configs(cls, config):
         return config
 
     @staticmethod
-    def _paths_to_code_from_fn_pointers(fn_pointers):
-        root_dir = rns_client.locate_working_dir()
-        file_path = fn_pointers[1].replace(".", "/") + ".py"
-        paths_to_code = [os.path.join(root_dir, file_path)]
-        return paths_to_code
-
-    def _reqs_to_list(self, env):
+    def _reqs_to_list(env):
```

Comment:
do we still need this if we are sending the env to the lambda where the Env object can handle the reqs internally?